Clinical-Genomics/cg
# Adds bioinformatic analysis info

This PR Adds the bait set to the report

How to test:
0. install on stage (add instructions on how to as needed)
1. us
1. cg upload delivery-report -p CUST-ID FAMILY-ID

Expected outcome:
The output should have the following properties:
* Application should have a header application
* Bionformatic analysis (e.g. MIP) should be the same as sample.data_analysis in status_db
* Bionformatic analysis (e.g. MIP) should have an header Bioinformatisk Analys
* Bait set should be present and be the same as sample.capture_kit in status db
